docs/API/API-socket-options.md

```diff
@@ -645,10 +645,17 @@ and the actual value for connected sockets.
 | ---------------- | ----- | -------- | ---------- | ------ | -------- | ------ | --- | ------ |
 | `SRTO_IPV6ONLY`  | 1.4.0 | pre-bind | `int32_t`  |        | (system) | -1..1  | RW  | GSD    |
 
-Set system socket flag `IPV6_V6ONLY`. When set to 0 a listening socket binding an
-IPv6 address accepts also IPv4 clients (their addresses will be formatted as
-IPv4-mapped IPv6 addresses). By default (-1) this option is not set and the
-platform default value is used.
+Set system socket option level `IPPROTO_IPV6` named `IPV6_V6ONLY`. This is meaningful
+only when the socket is going to be bound to the IPv6 wildcard address `in6addr_any`
+(known also as `::`). In this case this option must be also set explicitly to 0 or 1
+before binding, otherwise binding will fail (this is because it is not possible to
+determine the default value of this above-mentioned system option in any portable or
+reliable way). Possible values are:
+
+* -1: default, which can be used in cases not affected by this flag
```

Comment:
```suggestion
* -1: use platform-default value (default)
```

I don't get this sentence:
", which can be used in cases not affected by this flag"

Reply:
Cases that are not affected by this flag are all cases of binding to anything else than `::`.

Reply:
```suggestion
* -1: default, which can be used in cases not affected by this flag (when not binding to IPv6 wildcard `::`)
```